sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/SideEffect.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk
+
+/* A side effect. */
+object SideEffect {
+
+  /**
+   * Create a side effect of the given service call.
+   *
+   * @param serviceCall
+   *   The service call to effect.
+   * @param synchronous
+   *   Whether this effect should be executed synchronously.
+   * @return
+   *   The side effect.
+   */
+  def of(serviceCall: ServiceCall, synchronous: Boolean): SideEffect = ???
```

Comment:
Is there a reason for not naming it `apply`?

Reply:
yes, me doing mechanical copy-and-paste. 

I will change

Reply:
done